python/cuml/decomposition/pca.pyx

```diff
@@ -25,6 +25,7 @@ import numpy as np
 import cupy as cp
 import cupyx
 import scipy
+import warnings
```

Comment:
this is no longer being used, right?

Reply:
Ah darn! You're absolutely right. Fixing that up now...